src/Psalm/Report/CodeClimateReport.php

```diff
@@ -27,38 +27,21 @@ public function create(): string
         $options = $this->pretty ? Json::PRETTY : Json::DEFAULT;
 
         $issues_data = array_map(
-            function (IssueData $issue): array {
-                /**
```

Comment:
did you remove the comment or rector did?

Reply:
Rector did that. I'll go over the changes and restore missing comments.

Reply:
Meanwhile, I also reported this on rector issue tracker: https://github.com/rectorphp/rector/issues/6916

Reply:
It turned out this was the only comment missing (the other one I restored in the initial commit because Psalm was freaking out).